autoscale/src/main/java/com/sequenceiq/periscope/monitor/evaluator/load/YarnResponseUtils.java

```diff
@@ -17,35 +17,32 @@ public class YarnResponseUtils {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(YarnResponseUtils.class);
 
-    public List<String> getYarnRecommendedDecommissionHostsForHostGroup(String clusterCrn, YarnScalingServiceV1Response yarnResponse,
-            Map<String, String> hostFqdnsToInstanceId, int maxAllowedDownScale, Optional<Integer> mandatoryDownScaleCount, Integer maxScaleDownStepSize) {
+    public List<String> getYarnRecommendedDecommissionHostsForHostGroup(YarnScalingServiceV1Response yarnResponse,
```

Comment:
This generally needs some changes to get rid of all the additional parameters passed in here. See comments in YarnInvocationService.

Reply:
Updated this